integration_tests/suite/test_calls.py

```diff
@@ -1827,6 +1827,150 @@ def test_create_call_auto_answer(self):
             ),
         )
 
+    def test_unregistered_main_line(self):
+        user_uuid = 'user-uuid'
+        my_new_call_id = new_call_id()
+        priority = 1
+        self.confd.set_users(
+            MockUser(
+                uuid='user-uuid', line_ids=['line-id'], tenant_uuid='the-tenant-uuid'
+            )
+        )
+        self.confd.set_lines(
+            MockLine(id='line-id', name='line-name', protocol=CONFD_SIP_PROTOCOL)
+        )
+        self.ari.set_originates(
+            MockChannel(id=my_new_call_id, connected_line_number='')
+        )
```

Comment:
why is a mock channel required in this test? The calld request fails to connect the call, so there is no channel created yet in this scenario.

Reply:
for the two tests that return an error, you can omit the mock channel. i understand they are useful for the others which successfully create the call.

Reply:
Good point, I'll get rid of them